app/javascript/modules/listingDetails/ListingDetailsPricingTable.tsx

```diff
@@ -160,14 +159,14 @@ const buildAccordions = (
 
         const responsiveTableHeaders = listingIsSale
           ? {
-              units: { name: "t.unitType" },
```

Comment:
This change doesn't seem related to the React 18 upgrade. I'm just now realizing that this PR combines two unrelated(?) Jira tickets, as indicated by the PR title.

Reply:
@kramduckner Can you comment on this? I think this is a botched merge and should be reverted, but I wanted to confirm